docs/UserGuide.md

```diff
@@ -486,24 +529,30 @@ Deletes the topic from the current subject, given the topic number.
 
 Format: `delete topic [TOPIC_NUMBER]`
 
-+ Deletes the topic with the index number of `[TOPIC_NUMBER]`.
-+ `[TOPIC_NUMBER]` must be a **positive integer**.
-+ Topic `[TOPIC_NUMBER]` must **already exist in the topic list**.
+- Deletes the topic with the index number of `[TOPIC_NUMBER]`.
+- The application will print the title of the topic you have deleted from the topic list, as well as the amount of topics you 
+have in the topic list on the command line.
+- `[TOPIC_NUMBER]` must be a **positive integer**.
```

Comment:
should the line:
Topic `[TOPIC_NUMBER]` must **already exist in the topic list**.
be added to be consistent with the rest of the UG?